src/lib/iframe-tools/identity-manager.js

```diff
@@ -87,12 +87,28 @@ export class IdentityManager extends IframeManager {
       return;
     }
 
-    debugLogger.log('[identity-manager:fetchIdentity] Fetch country request.');
-    this.fetchCountry().then((country) => {
+    let shouldReallyCheckCountry;
+    const localStorageKey = 'pp-sdk-shouldReallyCheckCountry';
+    const LS = window.localStorage;
+    if (LS.getItem(localStorageKey) === null) {
+      const randomChance = Math.floor(Math.random() * 100);
+      shouldReallyCheckCountry = randomChance < 50;
+      LS.setItem(localStorageKey, shouldReallyCheckCountry);
+    } else {
+      shouldReallyCheckCountry = LS.getItem(localStorageKey);
+    }
 
-      debugLogger.log('[identity-manager:fetchIdentity] Fetch identity request. Country:', country);
-      this.fetchUserInfo(country);
-    });
+    debugLogger.log(`[identity-manager:fetchIdentity] Fetching Identity`);
+
+    if (shouldReallyCheckCountry) {
+      this.fetchCountry().then((country) => {
+        debugLogger.log('[identity-manager:fetchIdentity] Fetch identity request. Country:', country);
+        this.fetchUserInfo(country);
+      });
+    } else {
+      debugLogger.log('[identity-manager:fetchIdentity] Fetch identity request. Country not fetched.');
+      this.fetchUserInfo('ALL');
+    }
```

Comment:
Any unit test additions needed to verify these functional changes?

Reply:
I don't think so. Our grand plan to remove country check at all. It is temporary solution, just to be sure we don't have any unexpected results